README.md

```diff
@@ -163,6 +163,14 @@ Mapping the constellation of [Clojure](https://en.wikipedia.org/wiki/Clojure)-li
 * Built-in gradual type system which allows for malli-style type annotations which result in static type analysis.
 * [Work in progress](https://jank-lang.org/progress/)
 
+### [Calcit](http://calcit-lang.org/)
+
+> heavily influenced by Clojure APIs, Macros, persistent data structure. previously comipling to Clojure.
```

Comment:
Hello, do you mind capitalizing the sentences and fixing the spelling of "comipling"?

Reply:
sure